packages/website/src/components/search/search.css

```
@@ -1,29 +1,81 @@
+  --ma-docsearch-button-key-active-color: white;
+  --ma-docsearch-button-key-focus-visible-background-color: black;
+  --ma-docsearch-button-key-focus-visible-color: white;
+  --ma-docsearch-button-padding-block-end: 12px;
+  --ma-docsearch-button-padding-block-start: 12px;
+  --ma-docsearch-button-padding-inline-end: 16px;
+  --ma-docsearch-button-padding-inline-start: 16px;
+
+  /* - */
+  --docsearch-subtle-color: var(--ma-docsearch-button-border-color);
+  --docsearch-muted-color: var(--utrecht-button-secondary-action-color);
+  --docsearch-logo-color: var(--utrecht-button-secondary-action-color);
+  --docsearch-icon-color: var(--utrecht-button-secondary-action-color);
+  --docsearch-highlight-color: var(--utrecht-button-secondary-action-color);
+  --docsearch-hit-highlight-color: var(--basis-color-accent-1-bg-default);
+
+  & .DocSearch-Input {
+    margin-inline: 1rem;
+  }
+}
+
+#docsearch {
+  align-items: center;
+  box-sizing: border-box;
   display: flex;
-  inline-size: 100%;
-  justify-content: end;
 
-  & .utrecht-textbox {
-    --utrecht-textbox-border-radius: var(--basis-border-radius-sm);
+  & .DocSearch-Button {
+    border-radius: var(--ma-docsearch-button-border-radius);
+    box-sizing: border-box;
+    height: unset;
+    padding-block-end: var(--ma-docsearch-button-padding-block-end);
+    padding-block-start: var(--ma-docsearch-button-padding-block-start);
+    padding-inline-end: var(--ma-docsearch-button-padding-inline-end);
+    padding-inline-start: var(--ma-docsearch-button-padding-inline-start);
 
-    display: none;
+    &:hover {
+      --ma-docsearch-button-key-background-color: var(--ma-docsearch-button-key-hover-background-color);
+      --ma-docsearch-button-key-color: var(--ma-docsearch-button-key-hover-color);
 
-    /* @FIXME: Value taken from Figma should be more descriptive */
-    min-inline-size: 304px;
-  }
+      background-color: var(--ma-docsearch-button-hover-background-color);
+      border-color: var(--ma-docsearch-button-hover-border-color);
+      color: var(--ma-docsearch-button-hover-border-color);
+    }
 
-  @container ma-search (width <= 25rem) {
-    & .utrecht-button {
-      --utrecht-button-secondary-action-border-color: transparent;
+    &:focus-visible {
+      --ma-docsearch-button-key-background-color: var(--ma-docsearch-button-key-focus-visible-background-color);
+      --ma-docsearch-button-key-color: var(--ma-docsearch-button-key-focus-visible-color);
     }
   }
 
-  @container ma-search (width > 25rem) {
-    & .utrecht-textbox {
-      display: unset;
-    }
+  & .DocSearch-Button-Placeholder {
+    font-family: var(--ma-docsearch-button-font-family);
+    font-weight: 700;
+  }
+
+  & .DocSearch-Button-Key {
+    --ma-focus-indicator-color: var(--ma-docsearch-button-key-focus-visible-color);
+
+    background-color: var(--ma-docsearch-button-key-background-color);
+    border: none;
+    color: var(--ma-docsearch-button-key-color);
```

Misschien system colors fallback voor inverse colors?

```suggestion
    background-color: var(--ma-docsearch-button-key-background-color, CanvasText);
    border: none;
    color: var(--ma-docsearch-button-key-color, Canvas);
```